.prettierrc

```diff
@@ -1,8 +1,8 @@
 {
-	"endOfLine": "lf",
```

Comment:
Do you have `"editor.tabSize": 2,` in your VS Code `settings.json`?

Reply:
tab size 4

Reply:
Change, please, for 2 and reformat one file. Had something happen?